### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by nicko-winner**
Add Toast component

### CODE REVIEW

File: src/Alert/index.js [hunk 1]
```diff
@@ -5,12 +5,12 @@ import useEventListener from '../hooks/useEventListener';
 const Alert = ({ onOpen, onClose, className, children, onDismiss, onSubmit, ...rest }) => {
   const hxRef = useEventListener({ onDismiss, onSubmit });
   return (
-    <>
+    <div>
```
Comment (nicko-winner) [inline src/Alert/index.js:8]: notices `<>` elements don't fix the disappearing react element error.

Comment (100stacks) [reply src/Alert/index.js:8]: @nicko-winner, sorry what's this error again?  Does the `<>` cause the error on render?

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/Alert/index.js (hunk 1 above)
Comment (nicko-winner) [reply src/Alert/index.js:8]: The error happens after helix removes the element from the DOM (by clicking dismiss), and you change pages in storybook. This is the error:

Comment (100stacks) [reply src/Alert/index.js:8]: Ok thanks for the explanation.  This helps alot. 👍

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**